src/app/(nextjs_migration)/(guest)/breaches/layout.tsx

```diff
@@ -0,0 +1,130 @@
+
+  return (
+    <div data-partial="allBreaches">
+      <Script src="/nextjs_migration/client/js/allBreaches.js" />
+      <div id="breaches-loader" className="ab-bg breaches-loader"></div>
+      <main>
+        <div className="all-breaches-front-matter">
+          <header className="all-breaches-header">
+            <div>
+              <h1>{l10n.getString("all-breaches-headline-2")}</h1>
+              <p>{l10n.getString("all-breaches-lead")}</p>
+            </div>
+          </header>
+          <form className="all-breaches-filter" autoComplete="off">
+            <label htmlFor="breach-search">
+              <svg
+                role="img"
+                aria-label={l10n.getString("search-breaches")}
+                className="search-icon"
+                xmlns="http://www.w3.org/2000/svg"
+                width="16"
+                height="16"
+                viewBox="0 0 16 16"
+              >
+                <title>{l10n.getString("search-breaches")}</title>
+                <path
+                  fill="#5b5b66"
+                  d="M15.707 14.293l-4.822-4.822a6.019 6.019 0 1 0-1.414 1.414l4.822 4.822a1 1 0 0 0 1.414-1.414zM6 10a4 4 0 1 1 4-4 4 4 0 0 1-4 4z"
+                ></path>
+              </svg>
+            </label>
+            <input id="breach-search" type="search" autoFocus></input>
+          </form>
+        </div>
+        <section>
+          <div>
+            <div className="card-container">
+              {allBreaches
+                .filter((a) => !a.IsSensitive)
+                .sort((a, b) =>
+                  new Date(a.AddedDate).getTime() <
+                  new Date(b.AddedDate).getTime()
+                    ? 1
+                    : -1
+                )
+                .map((breach) => (
+                  <BreachCard
+                    key={breach.Name + breach.Domain}
+                    breach={breach}
+                    logos={breachLogos}
+                  />
+                ))}
+            </div>
+          </div>
+          <div className="row flx-col">
+            {/* TODO span id="no-results-blurb" className="no-results-blurb">{l10n.getString('no-results-blurb')}</span> */}
+          </div>
+        </section>
+      </main>
+    </div>
+  );
+}
+
+function BreachCard(props: { breach: any; logos: any }) {
+  const l10n = getL10n();
+
+  return (
+    <a href={`/breach-details/${props.breach.Name}`} className="breach-card">
+      <h3>
+        <span className="logo-wrapper">
+          <BreachLogo breach={props.breach} logos={props.logos} />
+        </span>
+        <span>{props.breach.Title}</span>
+      </h3>
+      <div className="breach-main">
+        <dl>
+          <div>
+            <dt>{l10n.getString("breach-added-label")}</dt>
+            <dd>
+              {new Date(props.breach.AddedDate).toLocaleString(getLocale(), {
+                year: "numeric",
+                month: "long",
+                day: "numeric",
+              })}
```

Comment:
I feel like the date localizer should possibly be in a helper. I think I've seen this pattern 2-3+ other places in the codebase too, so might be nice to just have a single helper that formats a date using a specified locale.

```sh
git grep -En "day: ['\"]numeric['\"]" | cat

public/nextjs_migration/client/js/scan.js:118:  newCard.querySelector('.exposure-scan-breach-added dd').textContent = new Date(breach.AddedDate).toLocaleString(navigator.languages, { year: 'numeric', month: 'long', day: 'numeric' })
src/app/(nextjs_migration)/(guest)/breaches/layout.tsx:100:                day: "numeric",
src/app/(nextjs_migration)/(guest)/breaches/page.tsx:77:            <dd>{new Date(props.breach.AddedDate).toLocaleString(getLocale(), { year: 'numeric', month: 'long', day: 'numeric' })}</dd>
src/client/js/partials/exposureScan.js:118:  newCard.querySelector('.exposure-scan-breach-added dd').textContent = new Date(breach.AddedDate).toLocaleString(navigator.languages, { year: 'numeric', month: 'long', day: 'numeric' })
src/utils/formatDate.js:7:  const options = { year: 'numeric', month: 'long', day: 'numeric' }
src/views/partials/allBreaches.js:19:          <dd>${new Date(breach.AddedDate).toLocaleString(getLocale(), { year: 'numeric', month: 'long', day: 'numeric' })}</dd>
src/views/partials/breachDetail.js:152:          breachDate: data.breach.BreachDate.toLocaleString(getLocale(), { year: 'numeric', month: 'long', day: 'numeric' }),
src/views/partials/breachDetail.js:154:          addedDate: data.breach.AddedDate.toLocaleString(getLocale(), { year: 'numeric', month: 'long', day: 'numeric' })
```

Actually, we might already have said helper function in https://github.com/mozilla/blurts-server/blob/main/src/utils/formatDate.js.

https://github.com/mozilla/blurts-server/blob/0bb98e8778732211d4e3f3737a5a413a20983821/src/utils/formatDate.js#L5-L13